### PR TITLE
Moebius transformations

### PR DESCRIPTION
**Pull request by HechtiDerLachs**
@simonbrandhorst : I wrapped some of the functionality we talked about into internal methods. Things seem to run rather smoothly now.

### CODE REVIEW

**Comment by HechtiDerLachs** [comment]
Tests pass. Ready for merge from my side.

File: experimental/Schemes/elliptic_surface.jl [hunk 1]
```diff
@@ -2229,3 +2242,250 @@ function _pushforward_section(
...
+  EE, _ = integral_model(EE)
+
+  # ...and back.
+  E2 = base_change(x->evaluate(x, gen(kkt)), EE)
+
+  @assert is_isomorphic(E, E2)
+  a, b, _ = rational_maps(isomorphism(E2, E))
+
+  eq_E = equation(E)
+  @show eq_E
```
Comment (simonbrandhorst) [inline experimental/Schemes/elliptic_surface.jl]:
```suggestion
```

File: experimental/Schemes/elliptic_surface.jl [hunk 2]
```diff
@@ -2229,3 +2242,250 @@ function _pushforward_section(
...
+  eq_E2 = equation(E2)
+  @show eq_E2
```
Comment (simonbrandhorst) [inline experimental/Schemes/elliptic_surface.jl]:
```suggestion
```

**Review by simonbrandhorst — APPROVED** [review]

**Comment by simonbrandhorst** [comment]
Thanks.

File: experimental/Schemes/elliptic_surface.jl (hunk 1 above)
Comment (HechtiDerLachs) [reply experimental/Schemes/elliptic_surface.jl]: Oops...

**Pull request merged**